crates/holochain/src/conductor/cell.rs

```diff
@@ -796,6 +796,8 @@ impl Cell {
                         |row| row.get(0),
                     )?;
 
+                    // TODO This flag is being set in the DHT database but the publish workflow reads from the authored
```

Comment:
Very bad unless I'm missing something here. Will be next on my list to address under workflow related changes

Reply:
Very bad indeed! We should do an audit of all the fields in the cell DB schema that aren't shared between authored and DHT dbs, there might be other mistakes like this.

Reply:
I think an audit is a good idea. I'm concerned by the fields that do overlap, it's simple enough to write queries that run against the authored or dht database. I understand managing some state as a table in the authored database but I do wonder if the schemas should be so similar that it's easy to mix them up.